### PR TITLE
Antrea cni pack

### PR DESCRIPTION
**Pull request by ritawatson**
This PR documents the Antrea CNI pack. Please note the following: 

- I added a couple parameters to the table that seem to apply to this pack. 
- I made the first listed parameter "required" because my understanding is that users can't use the pack without enabling nodIPAM.
- Roshani, please review all sections of the doc for accuracy. 

[**Preview**](https://deploy-preview-1141--docs-spectrocloud.netlify.app/integrations/antrea-cni)
[**Jira**](https://spectrocloud.atlassian.net/browse/PAC-155)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1141#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | f2b4ee6516b2c0d8afa6d8d5760f47c635b1dfa3 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1141/librarium_build/1628121563230375936) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1141#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-antrea-cni.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Antrea CNI
+
+Palette supports Antrea controller network interface (CNI) for Kubernetes clusters. Antrea  CNI enables each pod to have exclusive IP addresses from the subnet with direct accessibility.
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-antrea-cni.md:20]:
```suggestion
Palette supports Antrea controller network interface (CNI) for Kubernetes clusters. Antrea CNI enables each pod to have exclusive IP addresses from the subnet with direct accessibility. 
```

File: content/docs/06-integrations/00-antrea-cni.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Antrea CNI
+
+Palette supports Antrea controller network interface (CNI) for Kubernetes clusters. Antrea  CNI enables each pod to have exclusive IP addresses from the subnet with direct accessibility. 
+
+Antrea leverages Open vSwitch to implement pod networking and security features. Open vSwitch enables Antrea to implement Kubernetes network policies efficiently.
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-antrea-cni.md:22]:
```suggestion
Antrea leverages [Open vSwitch](https://www.openvswitch.org/) to implement pod networking and security features. Open vSwitch enables Antrea to implement Kubernetes network policies efficiently.
```

File: content/docs/06-integrations/00-antrea-cni.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+# Supported Versions
+
+**1.9.x**
+
+## Prerequisites
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-antrea-cni.md:28]:
```suggestion
# Prerequisites
```

Because we only have one element, these have to get raised by one level. Once we add another major version and add a tab, we decrease all of these by one.

File: content/docs/06-integrations/00-antrea-cni.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+    <WarningBox>
+
+    To avoid overlapping your pod network with any of your host networks, you should think of a suitable CIDR block to specify if you deploy a cluster using ``kubeadm init`` with ``--pod-network-cidr <cidr>`` or as a replacement in your network plugin's YAML.
+
+    </WarningBox>
+
+- The Open vSwitch kernel module must be present on every Kubernetes node.
+
+## Parameters
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-antrea-cni.md:45]:
```suggestion
# Parameters
```

File: content/docs/06-integrations/00-antrea-cni.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+| nodeIPAM:enable | This enables the integrated NodeIPAM controller within the Antrea controller. The default is `false`. | Y |
+| clusterCIDRs | CIDR ranges for pods in the cluster. The CIDRs could be either IPv4 or IPv6. You can specify up to one CIDR for each IP family. | N |
+| serviceCIDRv6 | IPv6 CIDR ranges reserved for Services. | N |
+| nodeCIDRMaskSizeIPv4 | Mask size for IPv4 Node CIDR in IPv4 or dual-stack cluster. | N |
+| nodeCIDRMaskSizeIPv6 | Mask size for IPv6 Node CIDR in IPv6 or dual-stack cluster. | N |
+| NodeIPAM | The feature toggle for ``antrea-controller``. The default is `false`. If you use CIDR ranges, set this to ``true``.  | N |
+| ServiceExternalIP | The feature toggle for ``antrea-agent`` and ``antrea-controller``. If you use the LoadBalancer service, set this to ``true``. | N |
+
+
+## Usage
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-antrea-cni.md:60]:
```suggestion
# Usage
```

File: content/docs/06-integrations/00-antrea-cni.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+Kubernetes network policies are supported by default.
+
+Antrea supports LoadBalancer services. Typically, implementing LoadBalancer services requires an external load balancer that is implemented by the Kubernetes Cloud Provider.
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-antrea-cni.md:64]:
```suggestion
Antrea supports LoadBalancer services. Typically, implementing LoadBalancer services requires an external load balancer that is implemented by the Kubernetes cloud provider. 
```
the cloud provider feels odd as a proper noun here when it's used to reference all cloud providers.

File: content/docs/06-integrations/00-antrea-cni.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+Antrea supports LoadBalancer services. Typically, implementing LoadBalancer services requires an external load balancer that is implemented by the Kubernetes Cloud Provider. 
+
+Antrea provides two options for supporting LoadBalancer services without using an external load balancer:
+- Using Antrea’s built-in external IP management for Services of type LoadBalancer.
+
+
+- Leveraging MetalLB.
+
+For detailed information, refer to Antrea’s [Service of type LoadBalancer](https://antrea.io/docs/v1.9.0/docs/service-loadbalancer) documentation.
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-antrea-cni.md:72]: There is also a docs page for using Antrea with MetalLB.  We should also incorporate that link, so maybe we should add these inline with the bullet?

https://antrea.io/docs/v1.9.0/docs/service-loadbalancer/#using-metallb-with-antrea

File: content/docs/06-integrations/00-antrea-cni.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+- Using Antrea’s built-in external IP management for Services of type LoadBalancer.
+
+
+- Leveraging MetalLB.
+
+For detailed information, refer to Antrea’s [Service of type LoadBalancer](https://antrea.io/docs/v1.9.0/docs/service-loadbalancer) documentation. 
+
+# Troubleshooting
+
+Ensure you have provided a non-overlapping IP address for your pod network  in the Kubernetes pack.
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-antrea-cni.md:76]: This feels odd, we should prefix the scenario for where this is the answer.  See the [Spectro Proxy](https://docs.spectrocloud.com/integrations/frp#troubleshooting) for an example of how this should look.

File: content/docs/06-integrations/00-antrea-cni.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+
+```tf
+data "spectrocloud_registry" "public_registry" {
+ name = "Public Repo"
+}
+
+data "spectrocloud_pack_simple" "antrea" {
+ name    = "antrea"
+ version = "1.9.0"
+ type = "network"
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-antrea-cni.md:91]:
```suggestion
 type = "helm"
```

**Review by karl-cardenas-coding — APPROVED** [review]
Besides the few suggestions, this PR looks great.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1141#pullrequestreview-1308200697" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1141#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-antrea-cni.md (hunk 1 above)
Comment (ritawatson) [reply content/docs/06-integrations/00-antrea-cni.md:20]: I don't see a change here, but I do see one further down, which I committed.

File: content/docs/06-integrations/00-antrea-cni.md (hunk 9 above)
Comment (ritawatson) [reply content/docs/06-integrations/00-antrea-cni.md]: Done

File: content/docs/06-integrations/00-antrea-cni.md [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+- Enable the ``enableNodeIPAM`` integrated NodeIPAM controller within the Antrea controller.
+- When deploying a cluster using kubeadm, specify the ``--pod-network-cidr <cidr>`` option and provide the IP address with the classless inter-domain routing (CIDR). For example: 
+
+    ``--pod-network-cidr=10.244.0.0/16``
+
+    <br />
+
+    <WarningBox>
+
+    To avoid overlapping your pod network with any of your host networks, you should think of a suitable CIDR block to specify if you deploy a cluster using ``kubeadm init`` with ``--pod-network-cidr <cidr>`` or as a replacement in your network plugin's YAML.
```
Comment (roshanirathi) [inline content/docs/06-integrations/00-antrea-cni.md:39]: For palette, podCIDR mentioned in the k8s values.yaml will always take preference. If you want to use the Antrea CIDRs, (only podCIDR works, service CIDR doesn’t) you need to empty values for pod and service CIDR in the k8s pack.

File: content/docs/06-integrations/00-antrea-cni.md [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+- Using Antrea’s built-in external IP management for Services of type LoadBalancer.
+
+
+- Leveraging MetalLB.
+
+For detailed information, refer to Antrea’s [Service of type LoadBalancer](https://antrea.io/docs/v1.9.0/docs/service-loadbalancer) documentation. 
+
+# Troubleshooting
+
+Ensure you have provided a non-overlapping IP address for your pod network  in the Kubernetes pack.
```
Comment (roshanirathi) [inline content/docs/06-integrations/00-antrea-cni.md:76]: IP address provided in k8s layer will always take preference. Refer to my previous comment

**Review by roshanirathi — CHANGES_REQUESTED** [review]

File: content/docs/06-integrations/00-antrea-cni.md (hunk 1 above)
Comment (karl-cardenas-coding) [reply content/docs/06-integrations/00-antrea-cni.md:20]: @ritawatson you have an extra whitespace at the beginning of the second sentence.

File: content/docs/06-integrations/00-antrea-cni.md (hunk 7 above)
Comment (ritawatson) [reply content/docs/06-integrations/00-antrea-cni.md:72]: Added link.

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | ff41fa258cd4cadbdca4a386c8295918ef0d8cab | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1141/librarium_build/1628469249011879936) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/06-integrations/00-antrea-cni.md (hunk 1 above)
Comment (ritawatson) [reply content/docs/06-integrations/00-antrea-cni.md]: fixed - thanks!

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 44bb2c21de73107cf52ddb204f1dc6cc695c53b2 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1141/librarium_build/1628470255321878528) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | eca482889607d14ad005596292ec808d947b60c1 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1141/librarium_build/1628471010367901696) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 8d1a0f71d379607745c63d7b973f5a7b8e576b7a | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1141/librarium_build/1628476295488737280) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 53f00bfa0aac5e51d4966c184a6cfe246db5f412 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1141/librarium_build/1628485228777443328) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/06-integrations/00-antrea-cni.md (hunk 8 above)
Comment (ritawatson) [reply content/docs/06-integrations/00-antrea-cni.md]: @roshanirathi Please review the revised Troubleshooting section for accuracy.

File: content/docs/06-integrations/00-antrea-cni.md (hunk 1 above)
Comment (ritawatson) [reply content/docs/06-integrations/00-antrea-cni.md]: @roshanirathi I added VMware to this sentence: "Palette supports Antrea controller network (CNI) for VMware Kubernetes clusters." 

Is this accurate?

File: content/docs/06-integrations/00-antrea-cni.md (hunk 10 above)
Comment (ritawatson) [reply content/docs/06-integrations/00-antrea-cni.md:51]: @roshanirathi Please review Troubleshooting section. You can see the doc at the Preview link in the PR Description.

File: content/docs/06-integrations/00-antrea-cni.md [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Antrea CNI
+
+Palette supports Antrea controller network interface (CNI) for VMware Kubernetes clusters. Antrea CNI enables each pod to have exclusive IP addresses from the subnet with direct accessibility.
```
Comment (ritawatson) [inline content/docs/06-integrations/00-antrea-cni.md:20]: @roshanirathi 'VMware' is added in the first sentence - is it accurate to do so?

File: content/docs/06-integrations/00-antrea-cni.md [hunk 13]
```diff
@@ -0,0 +1,137 @@
...
+
+# Supported Versions
+
+**1.9.x**
+
+<br />
+
+# Prerequisites
+
+- Enable the integrated NodeIPAM controller in the Antrea manifest: ``NodeIPAM:enable``.
```
Comment (ritawatson) [inline content/docs/06-integrations/00-antrea-cni.md:34]: @roshanirathi The prerequisite in line 34 is reworded. For consistency with 'Kubernetes manifest', 'Antrea controller' is changed here and throughout to say 'Antrea manifest'.

You can see the doc at the Preview link in the PR description.

File: content/docs/06-integrations/00-antrea-cni.md [hunk 14]
```diff
@@ -0,0 +1,137 @@
...
+
+# Parameters
+
+The Antrea CNI pack supports the following parameters.
+
+| Parameter | Description | Required (Y/N) |
+|-----------|-------------|---------|
+| nodeIPAM:enable | Enables the integrated NodeIPAM controller in the Antrea manifest. The default is `false`. | Y |
+| clusterCIDRs | CIDR ranges for pods in the cluster. The CIDRs can be either IPv4 or IPv6. You can specify up to one CIDR for each IP family. | N |
+| serviceCIDRv6 | IPv6 CIDR ranges reserved for Services. | N |
```
Comment (ritawatson) [inline content/docs/06-integrations/00-antrea-cni.md:68]: @roshanirathi You mentioned ``serviceCIDR`` doesn't work. Should it be removed from the parameters table?

File: content/docs/06-integrations/00-antrea-cni.md [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+
+- Leveraging MetalLB.
+
+For detailed information, refer to Antrea’s [Service of type LoadBalancer](https://antrea.io/docs/v1.9.0/docs/service-loadbalancer) documentation. 
+
+To learn more about using MetalLB, review [Using MetalLB with Antrea](https://antrea.io/docs/v1.9.0/docs/service-loadbalancer/#using-metallb-with-antrea). 
+
+<br />
+
+# Troubleshooting
```
Comment (ritawatson) [inline content/docs/06-integrations/00-antrea-cni.md:95]: @roshanirathi Troubleshooting section is updated. Please review for accuracy.

File: content/docs/06-integrations/00-antrea-cni.md [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+
+If routing problems occur or some hosts cannot communicate outside their subnet, this indicates overlapping IP addresses. 
+
+Ensure you have provided a non-overlapping IP address for your pod network in Palette's Kubernetes manifest using the ``podCIDR`` parameter. The CIDER IP specified with the ``podCIDR`` parameter in the Kubernetes manifest always takes precedence. 
+
+If you are using Antrea CIDRs and used the ``kubeadm init`` command with the ``--pod-network-cidr <cidr>`` attribute, ensure that you have done the following:
```
Comment (roshanirathi) [inline content/docs/06-integrations/00-antrea-cni.md:101]: `kubeadm init` command is used when we are manually creating a k8s cluster. And if that is case then we won't have a kubernetes manifest to remove values from. 
Instead we can reword this to -
If you wish to use Antrea CIDRs and have deployed a cluster using Palette, ensure that you have done the following:

File: content/docs/06-integrations/00-antrea-cni.md [hunk 17]
```diff
@@ -0,0 +1,137 @@
...
+# Supported Versions
+
+**1.9.x**
+
+<br />
+
+# Prerequisites
+
+- Enable the integrated NodeIPAM controller in the Antrea manifest: ``NodeIPAM:enable``.
+- When deploying a cluster using Palette, use the ``podCIDR`` parameter in the Pack section of the Kubernetes manifest. The CIDER IP specified with ``podCIDR`` always takes precedence.
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/06-integrations/00-antrea-cni.md:35]: Fixed.

File: content/docs/06-integrations/00-antrea-cni.md [hunk 18]
```diff
@@ -0,0 +1,137 @@
...
+
+To learn more about using MetalLB, review [Using MetalLB with Antrea](https://antrea.io/docs/v1.9.0/docs/service-loadbalancer/#using-metallb-with-antrea). 
+
+<br />
+
+# Troubleshooting
+
+If routing problems occur or some hosts cannot communicate outside their subnet, this indicates overlapping IP addresses. 
+
+Ensure you have provided a non-overlapping IP address for your pod network in Palette's Kubernetes manifest using the ``podCIDR`` parameter. The CIDER IP specified with the ``podCIDR`` parameter in the Kubernetes manifest always takes precedence.
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/06-integrations/00-antrea-cni.md:99]: Fixed

File: content/docs/06-integrations/00-antrea-cni.md (hunk 16 above)
Comment (ritawatson) [reply content/docs/06-integrations/00-antrea-cni.md:101]: Fixed

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | fb9878d4a3b6fe421ac1fd3bbce9fba45e1930b1 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1141/librarium_build/1630284332058808320) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/06-integrations/00-antrea-cni.md [hunk 19]
```diff
@@ -0,0 +1,137 @@
...
+
+    ``--pod-network-cidr=10.244.0.0/16``
+
+    <br />
+
+    <WarningBox>
+
+    The CIDER IP specified in Palette with the ``podCIDR`` parameter always takes precedence. 
+    
+    If you want to use Antrea CIDRs, you need to remove values for ``podCIDR`` in the Kubernetes manifest and any specified with ``serviceCIDR`` in the Antrea CNI manifest.
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/06-integrations/00-antrea-cni.md]: fixed.

File: content/docs/06-integrations/00-antrea-cni.md [hunk 20]
```diff
@@ -32,27 +32,35 @@ Antrea leverages [Open vSwitch](https://www.openvswitch.org/) to implement pod n
...
+    - Specify the ``--pod-network-cidr <cidr>`` option and provide the IP address with the CIDR. For example: 
 
     <br />
 
     ``--pod-network-cidr=10.244.0.0/16``
 
     <br />
 
-    <WarningBox>
+    - Remove any values for the ``podCIDR`` and ``serviceCIDR`` parameters in the Kubernetes manifest.
```
Comment (roshanirathi) [inline content/docs/06-integrations/00-antrea-cni.md:51]: As discussed, this is not applicable in case of `kubeadm init`. This can be removed from here.

**Comment by karl-cardenas-coding** [comment]
Gitleaks handled in the `master` branch. These are false positives.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1141#pullrequestreview-1308200697" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1141#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
